cmd/kpng/main.go

```diff
@@ -44,10 +44,16 @@ var (
 
 // main starts the kpng program by running the command sent by the user.  This is the entry point to kpng!
 func main() {
+	metricsPort := "9099"
```

Comment:
can u add a log statement in a follow on pr at some point saying metrics are on that port

Reply:
` I0407 00:32:48.665494       1 metrics.go:38] Starting metrics server at 0.0.0.0:9099  ` already shows up in the logs, is that good enough?

Reply:
In context:

```
kubectl logs -f -n kube-system kpng-9dzjs kpng
I0407 00:32:48.665334       1 main.go:78] exporting metrics to: 0.0.0.0:9099 
I0407 00:32:48.665494       1 metrics.go:38] Starting metrics server at 0.0.0.0:9099
...
```